install/kubernetes/cilium/values.yaml

```diff
@@ -164,7 +164,7 @@ azure:
   # clientSecret: 00000000-0000-0000-0000-000000000000
 
 # TODO: Add documentation
```

Comment:
If possible, maybe add docs here :thinking: , otherwise we can do it as part of #13496 as well :)

Reply:
The PR is about disabling the feature by default, and I intend to revert the related commit after the release is done. So I'd rather have the documentation added in another PR, if you don't mind.